test/abstract_store.ts

```diff
@@ -1,25 +1,29 @@
 import { IPublishPacket, IPubrelPacket } from 'mqtt-packet'
 import { IStore } from '../src/lib/store'
 import 'should'
+import { it, beforeEach, afterEach } from 'node:test'
 
 export default function abstractStoreTest(
 	build: (cb: (err?: Error, store?: IStore) => void) => void,
 ) {
 	let store: IStore
 
-	// eslint-disable-next-line
-	beforeEach(function (done) {
+	// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+	// @ts-expect-error
+	beforeEach((_ctx, done) => {
```

Comment:
This is the type error I mentioned above, use @ts-expect-error so in case this get fixed typescript will tell us